plugin.py

```diff
@@ -106,4 +106,10 @@ def on_start(self, window) -> bool:
         return server.ready
 
     def on_initialized(self, client) -> None:
-        pass  # extra initialization here.
+        client.on_notification(
+            'indexingStarted', lambda params: self.handle_indexing_status(started=True))
+        client.on_notification(
+            'indexingEnded', lambda params: self.handle_indexing_status(started=False))
+
+    def handle_indexing_status(self, started) -> None:
```

Comment:
Just a small nitpick :)

instead of `handle_indexing_status(self, started: bool)` maybe I would use `handle_indexing_status(self, msg: str)`
and I would just show the message. 
`sublime.status_message('Intelephense: Indexing {}'.format(msg))`

But the way you done it, also does the job fine.

Reply:
Sounds good :)